Backend/BikesRentalServer/Services/StationsService.cs

```diff
@@ -51,6 +54,8 @@ public ServiceActionResult<IEnumerable<Bike>> GetAllBikesAtStation(string id)
             var station = _dbContext.Stations.Include(s => s.Bikes).SingleOrDefault(s => s.Id == idAsInt);
             if (station is null)
                 return ServiceActionResult.EntityNotFound<IEnumerable<Bike>>("Station not found");
+            if(station.Status == BikeStationStatus.Blocked && _userContext.Role == UserRole.User)

```

Comment:
Powinno być sprawdzane w kontrolerze poprzez dodanie atrybutu [AdminAuthorization] (i być może [TechAuthorization] jeśli tech też to ma) do odpowiedniej metody StationsController

Reply:
Jest autoryzacja po stronie kontrolera, ale według ramla uzytkownik powinien móc pobrać rowery tylko z niezablokowanej stacji, a tech i admin zawsze i nie wiem jak to inaczej zrobić niż przez userContext

Reply:
Przyznam że ja też nie mam lepszego pomysłu w tej chwili. Zostaw jak jest, tylko wstaw spację przed nawiasem i zamień `==` na `is` (dla spójności, w innych miejscach tak już było to bądźmy konsekwentni)